src/man/pam_sss.8.xml

```diff
@@ -249,6 +252,19 @@ auth sufficient pam_sss.so allow_missing_name
                     </para>
                 </listitem>
             </varlistentry>
+            <varlistentry>
+                <term>
+                    <option>force_second_factor</option>
+                </term>
+                <listitem>
+                    <para>
+                        For sshd, if you have enabled 2FA and want to require 
+                        that both factors are provided, versus providing them
+			all in the first factor, set this option. A NULL or
+		        empty second factor will cause login to fail.	
```

Comment:
Fix the indentation

Reply:
Now there's a different indentation problem. I think it's because you are mixing tabs and spaces.